itest/universe_test.go

```diff
@@ -110,7 +112,12 @@ func testUniverseSync(t *harnessTest) {
 			}
 		}()
 
-		srcRoot, ok := universeRoots.UniverseRoots[uniKey]
+		// Construct universe namespace.
+		proofType, err := tap.UnmarshalUniProofType(newRoot.Id.ProofType)
+		require.NoError(t.t, err)
+		uniNamespace := fmt.Sprintf("%s-%s", proofType, uniKey)
```

Comment:
Hmm, still thinking about if we want to also manifest the prefix on this level, which arguably could/should just be something internal. 

I think will get a better feel of how it all looks on the CLI.